backend/src/plugins/Automod/actions/addRoles.ts

```diff
@@ -49,6 +48,7 @@ export const AddRolesAction = automodAction({
           "**, **",
         )}**`,
       });
+      return;
```

Comment:
It should still continue. The roles that cannot be assigned are filtered from the ones that will be assigned
```suggestion
```